app/static/i18n/fr.json

```diff
@@ -0,0 +1,147 @@
+{
+    "STATUSBAR": {
+        "CLUSTER": "Grappe",
+        "VERSION": "Version",
+        "SNAPSHOT": "INSTANTANÉ",
```

Comment:
Pareil pour snapshot, je ne trouve pas que 'Instantané' explique bien le concept

Reply:
There are no official recommendation for the word `snapshot` however companies, the likes of Amazon of Microsoft routinely use `instantanés` in there official documentation and products.

Example here : https://docs.aws.amazon.com/fr_fr/AWSEC2/latest/UserGuide/ebs-creating-snapshot.html
Or there : https://docs.microsoft.com/fr-fr/virtualization/hyper-v-on-windows/user-guide/checkpoints

A simple Wikipedia-ing will get you the same usage:
https://fr.wikipedia.org/wiki/Instantané_(informatique)

To be more be specific some use `capture instantanée` to reflect both `snap` and `shot` separately.
That said, I hear that others, such as VMware, do not; and use the word `snapshot` instead.